### PR TITLE
remove_script: remove --force flag, auth with management key

### PR DESCRIPTION
**Pull request by justprosh**

### CODE REVIEW

File: particle-closures/src/host_closures.rs [hunk 1]
```diff
@@ -220,13 +225,14 @@ impl<C: Clone + Send + Sync + 'static + AsRef<KademliaApi> + AsRef<ConnectionPoo
     fn remove_script(&self, args: Args, params: ParticleParameters) -> Result<JValue, JError> {
         let mut args = args.function_args.into_iter();
 
+        if params.init_user_id != self.management_peer_id {
```
Comment (DieMyst) [inline particle-closures/src/host_closures.rs:228]: you can add a test here to check that the logic is working

Comment (folex) [inline particle-closures/src/host_closures.rs:228]: Wait, why deny? Anyone should be able to remove scripts they own. management_peer_id can remove scripts from anyone, that is.

File: particle-node/tests/script_storage.rs [hunk 2]
```diff
@@ -243,9 +246,12 @@ fn autoremove_failed() {
 fn remove_script_unauth() {
     let swarms = make_swarms(1);
 
-    let mut client = ConnectedClient::connect_to(swarms[0].multiaddr.clone())
-        .wrap_err("connect client")
-        .unwrap();
+    let mut client = ConnectedClient::connect_to_with_peer_id(
+        swarms[0].multiaddr.clone(),
+        Some(swarms[0].keypair.clone()),
```
Comment (folex) [inline particle-node/tests/script_storage.rs:251]: we need to check that owner of the script can remove the script

**Review by folex — CHANGES_REQUESTED** [review]

File: particle-node/tests/script_storage.rs [hunk 3]
```diff
@@ -57,9 +57,12 @@ fn stream_hello() {
 fn remove_script() {
     let swarms = make_swarms(1);
 
-    let mut client = ConnectedClient::connect_to(swarms[0].multiaddr.clone())
-        .wrap_err("connect client")
-        .unwrap();
+    let mut client = ConnectedClient::connect_to_with_peer_id(
```
Comment (folex) [inline particle-node/tests/script_storage.rs:60]: let's rename to `connect_as_owner`

File: particle-node/tests/script_storage.rs [hunk 4]
```diff
@@ -57,9 +57,12 @@ fn stream_hello() {
...
+        swarms[0].multiaddr.clone(),
+        Some(swarms[0].keypair.clone()),
```
Comment (folex) [inline particle-node/tests/script_storage.rs:62]: let's rename `ConnectedSwarm::keypair` to `management_keypair`

File: crates/test-utils/src/misc.rs [hunk 5]
```diff
@@ -200,7 +200,7 @@ where
                 peer_id,
                 multiaddr,
                 tmp_dir,
-                keypair,
+                management_keypair: keypair,
```
Comment (folex) [inline crates/test-utils/src/misc.rs:203]: you can rename `keypair` to avoid `:`

**Review by folex — APPROVED** [review]

**Pull request merged**